index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header id="cabecalho">
+      <h1 id="titulo">Lessons Learned por Daniel Ribeiro</h1>
+      <nav>
+        <ul>
+          <li><a href="#about-me">Sobre</a></li>
+          <li><a href="#lessons">Habilides</a></li>
```

Comment:
Opa! Acho que aqui estava faltando uma sílaba em "habilidades".
```suggestion
          <li><a href="#lessons">Habilidades</a></li>
```